src/components/icon/icon.component.js

```diff
@@ -97,6 +99,10 @@ Icon.propTypes = {
   mr: PropTypes.number,
   /** Margin left, given number will be multiplied by base spacing unit (8) */
   ml: PropTypes.number,
+  /** Aria label for accessibility purposes */
+  ariaLabel: PropTypes.string,
```

Comment:
Can you add these props into `icon.d.ts` please (you might need to create the definition files here)

Reply:
This still needs to be done I think